website/site/content/docs/updating-the-cms.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Updating the CMS
+position: 24
+---
+
+# How updates to Netlify CMS are done
```

Comment:
I think that this should be the same as the page title: "Updating the CMS".

Reply:
Done